test/server.js

```diff
@@ -113,6 +113,50 @@ app.get('/data/noMapMeta', (req, res, next) => {
     }
   );
 });
+app.get('/data/query/map', (req, res, next) => {
```

Comment:
I see that some of these are configured to simulate a badly-configured server that serves the wrong mime-type, but we could take advantage also of the abiility to map a file that has a particular file type e.g. `foo.geojson` to the correct content-type header response.  See lines 7-13 of server.js